cmd/core-command/res/configuration.toml

```diff
@@ -48,11 +48,11 @@ Host = 'localhost'
 Port = 8200
 # Use the core-meta data secrets due to core-command using core-meta-data's database for persistance.
 Path = '/v1/secret/edgex/metadata/'
-Protocol = 'https'
-RootCaCertPath = '/vault/config/pki/EdgeXFoundryCA/EdgeXFoundryCA.pem'
-ServerName = 'localhost'
+Protocol = 'http'
+RootCaCertPath = ''
+ServerName = ''
```

Comment:
@tonyespy this `ServerName` is only being used when there is TLS communication to Vault as of today.  Since we are removing Vault TLS, so we can blank all them out.

Reply:
OK, thanks for clarifying.